src/mlpack/methods/ann/layer/isrlu.hpp

```diff
@@ -0,0 +1,138 @@
+/**
+ * @file methods/ann/layer/isrlu.hpp
+ * @author Abhinav Anand
+ *
+ * Definition of the ISRLU activation function as described by Jonathan T. Barron.
+ *
+ * For more information, read the following paper.
+ *
+ * @code
+ * @article{
+ *   author  = {Carlile, Brad and Delamarter, Guy and Kinney, Paul and Marti,
+ *              Akiko and Whitney, Brian},
```

Comment:
It _looks_ like mlpack prefers to have lists of authors in First Last, First Last ... format. I think that may help with readability.

Reply:
So shall i remove `and` from the names.

Reply:
A suggestion, if you find it helpful/to your liking
```suggestion
 *   author  = {Brad Carlile, Guy Delamarter, Paul Kinney, Akiko Marti,
 *              and Brian Whitney},
```